install/install.sh

```diff
@@ -180,6 +180,7 @@ installFile() {
         runAsRoot rm "$DAPR_CLI_FILE"
     fi
     chmod o+x $tmp_root_dapr_cli
+    mkdir -p $DAPR_INSTALL_DIR
```

Comment:
Check if it does not exist and then create it please

Reply:
Doesn't `mkdir -p` already take care of it?